### PR TITLE
Updated BRBINFO-4-1 test after timeoutMs added

### PR DESCRIPTION
**Pull request by tehampson**
Implements test plan update from https://github.com/CHIP-Specifications/chip-test-plans/pull/4547

### CODE REVIEW

File: src/python_testing/TC_BRBINFO_4_1.py [hunk 1]
```diff
@@ -30,6 +30,7 @@
```
Comment (tehampson) [inline src/python_testing/TC_BRBINFO_4_1.py:30]: Blocked on https://github.com/CHIP-Specifications/chip-test-plans/pull/4547 landing first

File: src/python_testing/TC_BRBINFO_4_1.py [hunk 2]
```diff
@@ -30,6 +30,7 @@
 
 import chip.clusters as Clusters
```
Comment (tehampson) [inline src/python_testing/TC_BRBINFO_4_1.py:31]: Blocked on https://github.com/project-chip/connectedhomeip/pull/35174 landing first

Comment (tehampson) [inline src/python_testing/TC_BRBINFO_4_1.py:31]: Blocked on code gen landing first https://github.com/project-chip/connectedhomeip/pull/35154/

**Review by cecille — APPROVED** [review]

File: src/python_testing/TC_BRBINFO_4_1.py [hunk 3]
```diff
@@ -57,14 +58,25 @@ def desc_TC_BRBINFO_4_1(self) -> str:
...
+            TestStep("2", "Setting up subscribe to ActiveChange event"),
+            TestStep("3", "Check TimeoutMs too low fails"),
+            TestStep("4", "Check TimeoutMs too high fails"),
+            TestStep("5", "Check KeepActive successful with valid command parameters lowest possible TimeoutMs"),
+            TestStep("6", "Validate previous command results in ActiveChanged event shortly after ICD device checks-in"),
+            TestStep("7", "Check KeepActive successful with valid command parameters highest possible TimeoutMs"),
+            TestStep("8", "Validate previous command results in ActiveChanged event shortly after ICD device checks-in"),
+            TestStep("9", "Send multiple KeepActive commands during window where ICD device will not check in"),
+            TestStep("10", "Validate previous command results in single ActiveChanged event shortly after ICD device checks-in"),
+            TestStep("11", "Validate we recieved no additional ActiveChanged event after subsequent ICD check in"),
```
Comment (saurabhst) [inline src/python_testing/TC_BRBINFO_4_1.py:72]:
```suggestion
            TestStep("11", "Validate we received no additional ActiveChanged event after subsequent ICD check in"),
```

File: src/python_testing/TC_BRBINFO_4_1.py [hunk 4]
```diff
@@ -57,14 +58,25 @@ def desc_TC_BRBINFO_4_1(self) -> str:
...
+            TestStep("10", "Validate previous command results in single ActiveChanged event shortly after ICD device checks-in"),
+            TestStep("11", "Validate we received no additional ActiveChanged event after subsequent ICD check in"),
+            TestStep("12", "Send KeepActive command with shortest TimeoutMs value while TH_ICD is prevented from sending check-ins"),
+            TestStep("13", "TH allows TH_ICD to resume sending check-ins after timeout should have expired"),
+            TestStep("14", "Wait for TH_ICD to check into TH twice, then confirm we have had no new ActiveChanged events reported from DUT"),
+            TestStep("15", "Send KeepActive command with shortest TimeoutMs value while TH_ICD is prevented from sending check-ins"),
+            TestStep("16", "Wait 15 seconds then send second KeepActive command with double the TimeoutMs value of the previous step"),
+            TestStep("17", "TH allows TH_ICD to resume sending check-ins after timeout from step 15 expired but before second timeout from step 16 still valid"),
+            TestStep("18", "Wait for TH_ICD to check into TH, then confirm we have recieved new event from DUT")]
```
Comment (saurabhst) [inline src/python_testing/TC_BRBINFO_4_1.py:79]:
```suggestion
            TestStep("18", "Wait for TH_ICD to check into TH, then confirm we have received new event from DUT")]
```

**Review by saurabhst — APPROVED** [review]

**Pull request merged**